site/_docs/variables.md

```diff
@@ -130,6 +130,14 @@ following is a reference of the available data.
       </p></td>
     </tr>
     <tr>
+      <td><p><code>site.html_files</code></p></td>
+      <td><p>
+
+        A list of all static HTML files.
```

Comment:
This might be a little vague. What do you think about `A list of all static files which end in .html`?